### PR TITLE
SPLASH-34: Album screen UI

### PR DESCRIPTION
**Pull request by Pepegaga**

### CODE REVIEW

File: src/app/home/routing.tsx [hunk 1]
```diff
@@ -10,7 +10,7 @@ const Stack = createStackNavigator<HomeParamList>();
 
 export const HomeStackComponent: React.FC = () => {
     return (
-        <Stack.Navigator headerMode={'none'}>
+        <Stack.Navigator headerMode={'none'} initialRouteName={'HomeAlbumsScreen'}>
```
Comment (TheLonelyAstronaut) [inline src/app/home/routing.tsx:16]: Remove this before pushing final commit, we don't want to see this in develop branch

File: src/app/music-data/components/album.component.tsx [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+export const AlbumWrapper = styled.TouchableOpacity`
+    margin-top: ${(props) => props.theme.spacer * 2};
+    width: ${DEVICE_SIZE.width};
+    height: ${DEVICE_SIZE.height * 0.08};
+    background-color: ${(props) => props.theme.colors.screenBackground};
+    flex-direction: row;
+`;
+export const AlbumImage = styled.Image`
+    height: ${DEVICE_SIZE.height * 0.08};
```
Comment (TheLonelyAstronaut) [inline src/app/music-data/components/album.component.tsx:15]: use Image from /ui

File: src/app/music-data/reducers.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+// import { createReducer } from 'typesafe-redux-helpers';
+
+import { Album } from '../../mocks/albums';
+import { artists } from '../../mocks/artists';
+import { Track } from '../../types/music';
+
+export type Artist = {
```
Comment (TheLonelyAstronaut) [inline src/app/music-data/reducers.ts:7]: You already have type Artist in another PR, we will create single type later

File: src/app/music-data/screens/albums-screen.component.tsx [hunk 4]
```diff
@@ -1,10 +1,40 @@
-import React from 'react';
-import { View, Text } from 'react-native';
+import React, { useCallback } from 'react';
+import { FlatList } from 'react-native';
+import Icon from 'react-native-vector-icons/dist/Ionicons';
+import { useTheme } from 'styled-components';
+import styled from 'styled-components/native';
+
+import { albums } from '../../../mocks/albums';
```
Comment (TheLonelyAstronaut) [inline src/app/music-data/screens/albums-screen.component.tsx:7]: Stop using this. Now you should pass params to the screen in initialScreenOptions prop of Stack.Screen component in routing.

File: src/app/music-data/screens/albums-screen.component.tsx [hunk 5]
```diff
@@ -1,10 +1,40 @@
...
+export const AlbumsScreenComponent: React.FC<MusicDataProps> = (props: MusicDataProps) => {
+    const theme = useTheme();
 
-export const AlbumsScreenComponent: React.FC = () => {
     return (
-        <View>
-            <Text>Albums screen</Text>
-        </View>
+        <Container>
+            <BackButtonContainer
```
Comment (TheLonelyAstronaut) [inline src/app/music-data/screens/albums-screen.component.tsx:24]: You really should create back-button-component in /ui folder and use it in all screens

File: src/mocks/albums.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
+import { Track } from '../types/music';
+
+import { tracks } from './tracks';
+
+export interface Album {
+    data: Track[];
+    year: string;
+    img: string;
```
Comment (TheLonelyAstronaut) [inline src/mocks/albums.ts]: cover or artwork, not img

**Comment by Pepegaga** [comment]
All this was refactored in another branch by mistake, but everything is ok

**Pull request merged**